app/models/package_manager/packagist.rb

```diff
@@ -66,31 +68,27 @@ def self.deprecation_info(name)
     end
 
     def self.mapping(raw_project)
-      return nil unless raw_project["versions"].any?
-
-      # for version comparison of php, we want to reject any dev versions unless
-      # there are only dev versions of the project
-      versions = raw_project["versions"].values.reject { |v| v["version"].include? "dev" }
-      versions = raw_project["versions"].values if versions.empty?
-      # then we'll use the most recently published as our most recent version
-      latest_version = versions.max_by { |v| v["time"] }
+      return nil unless raw_project.any?
+
+      latest_version = raw_project
+        .max_by { |v| v["time"] } # then we'll use the most recently published as our most recent version
+
+      return if latest_version.nil?
+
       {
         name: latest_version["name"],
         description: latest_version["description"],
-        homepage: latest_version["home_page"],
+        homepage: latest_version["homepage"],
         keywords_array: Array.wrap(latest_version["keywords"]),
         licenses: latest_version["license"].join(","),
-        repository_url: repo_fallback(raw_project["repository"], latest_version["home_page"]),
+        repository_url: repo_fallback(latest_version["source"].first&.fetch("url"), latest_version["homepage"]),
```

Comment:
In the example file they have on their API docs site it looks like this will link to the `.git` url like `https://github.com/Seldaek/monolog.git`. Do you know if that URL gets normalized and removes the `.git` on the end?

Reply:
Ah, good find! It does not get normalized, so I'll strip that

Reply:
Woops, nevermind, the `repo_fallback` method actually does `URLParser.try_all`, which cleans any VCS urls.